decidim-elections/config/locales/en.yml

```diff
@@ -942,20 +942,27 @@ en:
         menu:
           polling_officer_zone: Polling Officer zone
         polling_officers:
-          show:
-            actions:
-              count_votes: Count the votes in the ballot box
-              identify_person: Identify and verify a person
+          index:
             polling_officer_role_description: You have been assigned to act as a Polling Station Officer (President or Manager) in some of the elections celebrated in this platform.
-            polling_stations:
-              list:
-                actions: Actions
-                address: Address
-                name: Name
-                role: Your role
-                voting: Voting
+            polling_station:
+              address: Address
+              count_votes: Count votes
+              election: Election name
+              identify_person: Identify a person
+              name: Polling station name
               no_polling_stations: You are not assigned to any Polling Station yet.
+              role: Your role
               title: Polling Stations
+              voting: Voting space
+        results:
+          create:
+            error: An error ocurred creating the results, please try again later.
```

Comment:
```suggestion
            error: An error occurred creating the results, please try again later.
```

Reply:
Good catch, I'll fix other _occurrences_ found in this file 😁. I'll also fix it in the following PR (as commented above).

Reply:
The following PR is #7804 😄